src/main/res/xml/whereyougo_preferences_appearance.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:icon="@drawable/ic_action_picture"
+    android:key=""
+    android:persistent="false"
+    android:title="@string/pref_appearance">
+
+    <PreferenceCategory
+        android:key=""
```

Comment:
You have added an `android:key` for other `PreferenceCategory`.
If it make sense to do it so then maybe also here?

Reply:
Does make sense. Done via force-push.